docs/ScoreCard.md

```diff
@@ -74,7 +74,7 @@ Child element with attributes:
 
 <div style="overflow: auto;">
 
-| Attributes  | Description                                 | Required | Default |
+| Selector    | Description                                 | Required | Default |
```

Comment:
May want to change everything

Reply:
You're right, that was an accident change.  I think it makes sense to change `attributes` to `selector`, but not as a part of this PR.